src/components/Modals/PublishCollectionModal/PublishCollectionModal.tsx

```diff
@@ -62,93 +79,165 @@ export default class PublishCollectionModal extends React.PureComponent<Props, S
-                        Account
-                      </a>
-                    )
-                  }}
-                />
-              </small>
-            ) : null}
-          </>
-        )}
+              <Button className="proceed" primary fluid onClick={this.handleProceed} disabled={hasInsufficientMANA}>
+                {t('global.next')}
+              </Button>
+              {hasInsufficientMANA ? (
+                <small className="not-enough-mana-notice">
+                  <T
+                    id="publish_collection_modal.not_enogh_mana"
+                    values={{
+                      symbol: (
+                        <span>
+                          <Mana network={Network.MATIC} inline /> MANA
+                        </span>
+                      )
+                    }}
+                  />
+                  <br />
+                  <T
+                    id="publish_collection_modal.get_mana"
+                    values={{
+                      link: (
+                        <a href={env.get('REACT_APP_ACCOUNT_URL', '')} rel="noopener noreferrer" target="_blank">
+                          Account
+                        </a>
+                      )
+                    }}
+                  />
+                </small>
+              ) : null}
+            </>
+          )}
+        </Modal.Content>
       </>
     )
   }
 
   renderSecondStep = () => {
-    const { isLoading } = this.props
+    const { onClose } = this.props
 
     return (
       <>
-        {t('publish_collection_modal.first_paragraph')}
-        <div className="divider"></div>
-        {t('publish_collection_modal.second_paragraph')}
-        <div className="divider"></div>
-        {t('publish_collection_modal.third_paragraph')}{' '}
-        <a href="https://docs.decentraland.org/wearables/publishing-wearables" rel="noopener noreferrer" target="_blank">
-          {t('global.learn_more')}
-        </a>
-        <Button primary fluid onClick={this.handlePublish} loading={isLoading}>
-          {t('publish_collection_modal.publish')}
-        </Button>
+        <ModalNavigation title={t('publish_collection_modal.title_tos')} onClose={onClose} />
+        <Modal.Content className="second-step">
+          {t('publish_collection_modal.first_paragraph')}
+          <div className="divider"></div>
+          {t('publish_collection_modal.second_paragraph')}
+          <div className="divider"></div>
+          {t('publish_collection_modal.third_paragraph')}{' '}
+          <a href="https://docs.decentraland.org/wearables/publishing-wearables" rel="noopener noreferrer" target="_blank">
+            {t('global.learn_more')}
+          </a>
+          <Button primary fluid onClick={this.handleNextStep}>
+            {t('global.next')}
+          </Button>
+        </Modal.Content>
+      </>
+    )
+  }
+
+  renderStep = () => {
```

Comment:
Just for my own selfish OCD, could we move this method after `renderThirdStep` ?

Reply:
Done!